src/main/java/com/ibm/watson/developer_cloud/natural_language_classifier/v1/NaturalLanguageClassifier.java

```diff
@@ -46,12 +46,13 @@
  */
 public class NaturalLanguageClassifier extends WatsonService {
 
+  public static final String LANGUAGE_EN = "en";
+
   private static final String FORM_DATA_TRAINING_DATA = "form-data; name=\"training_data\"";
   private static final String PATH_CLASSIFIERS = "/v1/classifiers";
   private static final String TEXT = "text";
   private static final String PATH_CLASSIFY = "/v1/classifiers/%s/classify";
   private static final String LANGUAGE = "language";
-  public static final String LANGUAGE_EN = "en";
```

Comment:
It is a best code style to put public variable above private variable, also can be applied in public method and private method. The origin code inject a public static final variable into a lots of private static final variables, i think it is not a good style. what do you think?